src/lane_follow/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lane_follow)
+
+## Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS
+        roscpp
+        rospy
+        image_transport
+        cv_bridge
+        )
+
+find_package(OpenCV REQUIRED)
+
+find_package(sb_utils REQUIRED)
```

Comment:
nit-picky thing, but we can probably remove the blank lines between the `find_package` lines. I may need to fix this in the `IGVC-2017` repo as well.... :neutral_face: